.circleci/cimodel/data/binary_build_definitions.py

```diff
@@ -27,7 +27,7 @@ def gen_build_env_parms(self):
 
     def gen_docker_image(self):
         if self.gcc_config_variant == 'gcc5.4_cxx11-abi':
-            return miniutils.quote("pytorch/pytorch-binary-docker-image-ubuntu16.04:latest")
+            return miniutils.quote("pytorch/pytorch-binary-docker-image-ubuntu16.04:test")
```

Comment:
Going to go ahead and push the `test` tag to `latest` so we can replace this with `latest`

Reply:
we haven't tested the test tag with the other cuda versions though--that wouldn't be a problem right?

Reply:
Pretty sure your test PR had good enough coverage for all of them so I think it should be fine